src/main/java/LadderGame/Ladder.java

```diff
@@ -1,58 +1,90 @@
 package LadderGame;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Scanner;
+import java.util.stream.Collectors;
 
 public class Ladder {
-    private int numPeople;
-    private int height;
-    private String[][] ladder;
+    private int ladder_height;
+    private List<List<String>> ladder_set;
```

Comment:
List<String> 을 멤버로 가지는 다른 클래스를 가져도 좋겠네요. 
일급 컬렉션이라고 하는데 관련 키워드로 검색해 보세요.
```suggestion
    private List<List<String>> ladderSet;
```